app/Providers/ComposerServiceProvider.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\Facades\View;
+use Illuminate\Support\ServiceProvider;
+use App\Eloquent\Option;
+
+class ComposerServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        // View::composer(
+        //     '*',
+        //     'App\Http\ViewComposers\ProfileComposer'
+        // );
+
+        view()->share($this->option());
+    }
+
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        //
+    }
+
+    public function option()
+    {
+        $data['textFooters'] = Option::where(['key' => 'text_footer'])->get();
+        $data['banners'] = Option::where(['key' => 'banner'])->get();
+        $data['bannerBooks'] = Option::where(['key' => 'banner_book'])->first();
+        $data['apps'] = Option::where(['key' => 'app'])->get();
+        $data['textBanners'] = Option::where(['key' => 'text_banner'])->get();
+        $data['textApps'] = Option::where(['key' => 'app_text'])->get();
+        $data['contacts'] = Option::where(['key' => 'contact'])->get();
+        $data['address'] = Option::where(['key' => 'address'])->get();
+        $data['emails'] = Option::where(['key' => 'email'])->get();
+        return $data;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Providers/ComposerServiceProvider.php</br>**Line**: 49</br>Expected 1 newline at end of file; 0 found